Unity/Assets/YarnSpinner/Editor/YarnImporter.cs

```diff
@@ -87,7 +88,7 @@ private void ImportYarn(AssetImportContext ctx)
 
                 // Add this container to the imported asset; it will be
                 // what the user interacts with in Unity
-                ctx.AddObjectToAsset("Program", programContainer);
+                ctx.AddObjectToAsset("Program", programContainer, AssetDatabase.LoadAssetAtPath(AssetDatabase.GUIDToAssetPath("387416c1e6195934fbbc2dc50e24e8af"), typeof(Texture2D)) as Texture2D);
```

Comment:
What's the reasoning behind using a GUID to select the icon texture, as opposed to a search? (Not necessarily objecting, it's just that we're using path-based searches elsewhere.)

Reply:
Path-based searches break as soon as the user of this library moves it to a place preferred by the project's file structure. While you might be able to work around that one by doing the path based search on top of Yarn's base path, it would still break on actions like renaming the icon asset or renaming the icon directory so I considered the guid-based lookup to be more stable.